### DISCUSSION TITLE
Launching Application

### DISCUSSION THREAD

**Discussion by bfcox2000** [SamDel/ChromeCast-Desktop-Audio-Streamer#154] [Q&A]
I am very grateful to the creator of this application. I'm just having trouble with the application starting its streams.
My devices are discovered, but when I try to stream to them I often get the state "Launching Application", and things just stop there.
![image](https://github.com/SamDel/ChromeCast-Desktop-Audio-Streamer/assets/22846788/32443bba-4d9b-47a4-9ecf-1a0bfb9b23e2)

This happens most of the time, but at times the application works fine.
The firewall is turned off.
![image](https://github.com/SamDel/ChromeCast-Desktop-Audio-Streamer/assets/22846788/88d1aa1e-b4b1-4cef-a776-df000436f6c1)

**Comment by SamDel** [comment]
Can you post the log, then I can have a look?

Maybe it's a network configuration issue!? Something simular as [this one](https://github.com/SamDel/ChromeCast-Desktop-Audio-Streamer/issues/152#issue-1990874350). It looks like the device can't connect to the application.

**Comment by bfcox2000** [comment]
Sam,

Thank you, the reference you sent did the trick.
I had an extra interface that was confusing things, once I removed that the
problem went away

**Comment by SamDel** [comment]
👍

I'm closing this one.
